mlir/lib/Dialect/Linalg/Transforms/Transforms.cpp

```diff
@@ -434,8 +429,21 @@ FailureOr<LowerUnPackOpResult> linalg::lowerUnPack(RewriterBase &rewriter,
       RankedTensorType::Builder(packedTensorType).setShape(stripMinedShape);
   RankedTensorType collapsedType = tensor::CollapseShapeOp::inferCollapsedType(
       stripMinedTensorType, packingMetadata.reassociations);
+
+  // Get dynamic dims from input tensor in order of stripMinedTensor
```

Comment:
Not sure if I understand the comment. I would rephrase as: `Get dynamic dims from input tensor based on lastDimsToInsertPositionsPerm permutation`.